tik/tik.cpp

```diff
@@ -332,7 +337,7 @@ int main(int argc, char *argv[])
     }
     if (error)
     {
-        return EXIT_FAILURE;
```

Comment:
Don't we still want this to report as a failure?

Reply:
No. The new Dash-Automate flow pays attention to every return code. If an error is detected the pipeline is stopped. Thus, we need to use return codes to the OS to signify whether the tool succeeded in the defined-behavior sense. This error check is about a semantic error, and there needs to be signified in a different way.